buildspec-tests.yml

```diff
@@ -3,16 +3,12 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      python: 3.9
+      python: 3.11
     commands:
       - echo Installing dependencies...
-      - pip install -r requirements-test.txt
+      - pip install -r requirements/dev.txt
```

Comment:
What's the thinking behind the rename?

Reply:
These requirements are mainly for developers and isn't only for testing purposes. The `pre-commit` hook is a dev dependency, for example.

Reply:
I suggest we move to poetry anyway. What do you think?

Reply:
Yes - we have a JIRA ticket to address `poetry`: https://uktrade.atlassian.net/browse/GPM-530